src/runtime/graph/debug/graph_runtime_debug.cc

```diff
@@ -153,9 +153,10 @@ class GraphRuntimeDebug : public GraphRuntime {
     const TVMContext& ctx = data_entry_[entry_id(index, 0)]->ctx;
     TVMSynchronize(ctx.device_type, ctx.device_id, nullptr);
     auto op_tend = std::chrono::high_resolution_clock::now();
-    double op_duration =
-        std::chrono::duration_cast<std::chrono::duration<double> >(op_tend - op_tbegin).count();
-    return op_duration;
+    double op_duration_us =
+        std::chrono::duration_cast<std::chrono::duration<double> >(op_tend - op_tbegin).count() *
```

Comment:
ah...I think this was an incomplete migration in I forgot to fix when forward-porting. I think we should also either adjust [src/runtime/crt/common/crt_runtime_api.c](https://github.com/apache/tvm/blob/0230ddf3436f82a57719a8761fcccf39626ee2be/src/runtime/crt/common/crt_runtime_api.c#L487) or update the layers above this to respect seconds. I personally think seconds makes more sense as a unit for time expressed in double, but we should make that change in a separate PR. 

is this change needed to make your PR pass CI? if not, perhaps we should split into another or at least adjust both locations?

Reply:
Thanks! This does not block my PR, and I would remove this part from here.
I've opened a new issue #7207 to track this bug.